UIKitHometasks/UIKitHometasks/ViewController.swift

```diff
@@ -6,14 +6,96 @@
 //
 
 import UIKit
-
+/// controller
 class ViewController: UIViewController {
 
+    @IBOutlet weak var hiLabel: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        self.alertName(title: "Готов к игре?", message: "Введите имя", style: .alert)
     }
 
-
+    @IBAction func plusButton(_ sender: UIButton) {
+        alertSum(title: "Играем!", message: "Введи 2 числа", style: .alert)
+    }
+    
+    @IBAction func guessNumberButton(_ sender: UIButton) {
+        alertGuessNumber(tittle: "Играем!", message: "Загадал от 1 до 3", style: .alert)
+    }
+    
+    func alertName(title: String, message: String, style: UIAlertController.Style) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: style)
+        let action = UIAlertAction(title: "Давай!", style: .default) { _ in
+            let text = alertController.textFields?.first
+            guard let notOptName = text?.text else { return }
+            
+            if notOptName.isEmpty {
+                self.hiLabel.text = "Начинаем, незнакомец?"
+            } else {
+                self.hiLabel.text = "Начинаем, \(notOptName)?"
+            }
+        }
+        
+        alertController.addTextField { _ in
+        }
+        
+        alertController.addAction(action)
+        self.present(alertController, animated: true, completion: nil)
+    }
+    
+    func alertSum(title: String, message: String, style: UIAlertController.Style) {
+        let alertController = UIAlertController(title: title, message: message, preferredStyle: style)
+        alertController.addTextField { _ in
+        }
+        
+        alertController.addTextField { _ in
+        }
+        
+        let action = UIAlertAction(title: "Считай!", style: .default) { _ in
+            let firstNumber = Int(alertController.textFields?[0].text ?? "0")
+            let secondNumber = Int(alertController.textFields?[1].text ?? "0")
+            let secondAlertController = UIAlertController(title: "Good job!", message: "Result", preferredStyle: .alert)
+            let secondAction = UIAlertAction(title: "okay", style: .default, handler: nil)
+            secondAlertController.addTextField { _ in
+            }
+            
+            secondAlertController.textFields?[0].text = String((firstNumber ?? 0) + (secondNumber ?? 0))
```

Comment:
```suggestion
            secondAlertController.textFields?[0].text =( "/(firstNumber ?? 0)  /(secondNumber ?? 0)")
```
Используем интерполяцию вместо конкатенации